db/seeds/donations.rb

```diff
@@ -7,13 +7,13 @@
   event: Event.first
 ],
 [
-  donor: Donor.first,
+  donor: Donor.find(2),
```

Comment:
This is a bit fragile, because it assumes the `id` is always `2`. `Donor.second` would be better. 😀